bootstrap/cron.go

```diff
@@ -0,0 +1,50 @@
+package bootstrap
+
+import (
+	"go-google-scraper-challenge/helpers/log"
+	"go-google-scraper-challenge/lib/models"
+	"go-google-scraper-challenge/lib/services/scraper"
+	"time"
```

Comment:
```suggestion
	"time"

	"go-google-scraper-challenge/helpers/log"
	"go-google-scraper-challenge/lib/models"
	"go-google-scraper-challenge/lib/services/scraper"
```

Reply:
✅ Fixed in [aa81f63](https://github.com/carryall/go-google-scraper-challenge/pull/104/commits/aa81f6307917694b85c82c90a84e440f7a4f0f30)